### ISSUE TITLE
Chapter 3: Plausible Deniability - Manual Control - Turrets have no ammo and cannot be aimed

### ISSUE THREAD

**Issue by Kodiacicles** [RavienGaming/Reforged-Eden-2#215]
### Reforged Eden 2 Build Number

15

### Game mode

Survival, Single Player

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

Playing story mission "Plausible Deniability, chapter "Manual Control". After giving the engineer the components and talking to the captain, the mission requires controlling the ship's turrets manually to shoot down the drones. I can access the turrets, but cannot aim them (locked straight forward) and the turrets have no ammo. POI appears to have power, as the force field at the entrance ramp is working.  Issue happens when controlling either turret.
![no_ammo](https://github.com/user-attachments/assets/58c95133-e2b1-4021-9c28-565f453d0dc1)

### Error code from console or log (Optional)

None

### POI Name

R2BA_FreighterStory

### Playfield Name

Omicron

### Playfield Class

Arid Starter

### Star Name

Beta

### Star Class

FStartingSystem

### Mission Name

Manual Control

### Mission Objective

Destroy the Attacking Bombers

**Comment by VermillionX** [comment]
The POI is NPC-owned and using an admin core. All turrets on an AI-controlled ship/base have unlimited ammunition, regardless of what the ammo counter says.
The turrets are unlocked (since you have access of it) for player use. The only reason they wouldn't work is either a bug with the game, an edited config to disable admin protection, or the POI has been edited using creative/debug tools that are not part of survival.

**Comment by Kodiacicles** [comment]
OK, thank you. I haven't done any editing on the config or the POI. I'm not sure how to replicate it in the base game, but I was able to get things working by setting my own faction to Polaris and then accessing the POI and adding fuel to the tank. After that I was able to move and fire the turret, as well as having all the lights turn on and work- my original thought that the POI had power because the forcefield was working must be wrong, sorry. In the course of loading and unloading the game to get screenshots and check things, it's also spawned a lot more than 3 drones, which is more hilarious than problematic. 

Thanks again, and sorry for reporting what is most likely a base game bug to you.
![20240713135850_1](https://github.com/user-attachments/assets/634833c3-1d52-41c5-bb2a-8a6280c3b5b6)
